### PR TITLE
add easyconfig file for AMBER

### PR DESCRIPTION
**Pull request by boegel**
needs a little bit of cleanup

### CODE REVIEW

**Comment by boegel** [comment]
should be combined with https://github.com/hpcugent/easybuild-easyblocks/pull/491?

File: easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+easyblock = 'MakeCp'
+
+name = 'AMBER'
+version = "9.0.0"
+
+homepage = 'http://ambermd.org/amber.html'
+description = """AMBER is a Molecular Dynamics Simulation Software"""
+
+toolchain = {'name': 'ictce', 'version': '5.3.0', 'name': 'ictce'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb]: double `name`

File: easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+toolchainopts = {'usempi': True}
+
+#source_urls = ['http://www.example.com']
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb]: no source_url? It's commercial? Add a comment.

File: easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+sources = ['amber-%(version_major)s.tar.gz']
+
+patches = ["bugfix.all"]
+
+dependencies = []
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb]: drop

File: easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+#
+# author: Marios Constantinou (University of Cyprus)
+#
+
+easyblock = 'MakeCp'
```
Comment (boegel) [inline easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb]: this should go too, we should use the easyblock from https://github.com/hpcugent/easybuild-easyblocks/pull/491

**Comment by wpoely86** [comment]
Yes, this needs syncing with the block

**Comment by boegel** [comment]
I'm closing this as is, basically obsolete now #3200 has been merged in

**Pull request closed without merging**